nntrainer/compiler/compiler.h

```diff
@@ -0,0 +1,209 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2021 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file compiler.h
+ * @date 01 April 2021
+ * @brief NNTrainer compiler that reads and generates executable graph
+ * @details
+ * Graph is convertible either to iostream, representation, executable by
```

Comment:
add some scenarios of iostream like tflite format, ini, etc.

Reply:
Updated